sdk/purview/purview-datamap-rest/samples-dev/typedefs.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how get a list of typedefs
+ *
+ * @summary gets a list of typedefs for entities
+ * @azsdk-weight 40
+ */
+
+import PurviewDataMap from "@azure-rest/purview-datamap";
```

Comment:
I notice that there is only one sample, do you think this is necessary for customers?

Reply:
This is only for the first beta SDK. We will have follow-up items to provide more SDK examples based on the customer usage on SDK.